src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,6 @@
+%normalize {
+  font-family: roboto, sans-serif;
+  margin: 0;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.